### DISCUSSION TITLE
Is Apple Silicon supported by MiKTeX or does it run via Rosetta 2?

### DISCUSSION THREAD

**Discussion by veraverto** [MiKTeX/miktex#1310] [Q&A]
Hello everyone,

I've recently upgraded to an Apple silicon laptop and am considering to install MiKTeX instead of the giant package MacTeX. But before installing I was wondering whether MiKTeX also supports the new hardware natively like MacTeX or if would run via Rosetta 2.

Best!

**Comment by stefansundin** [comment]
I had the same question but it seems like unfortunately the answer is No.

I have always used MiKTeX by default, but after some research I ended up just using BasicTeX instead, which worked just fine for what I needed. https://www.tug.org/mactex/morepackages.html

There's also TeXstudio if you need a GUI. It requires BasicTeX or MacTeX. https://texstudio.sourceforge.net/

**Reply by stevenya97** [reply]
Is there anything like MiKTeX's auto download packages feature? I like that it keeps a light install but extends functionality on-demand and I can easily see which ones are being used in the GUI console. The full MacTex 6.4GB install is more daunting on base Macs with 256GB of storage.

**Reply by stefansundin** [reply]
I can't recall.
